### DISCUSSION TITLE
Internal Server Error when embedding videos

### DISCUSSION THREAD

**Discussion by CanEkizoglu** [requarks/wiki#5512] [Help / Questions]
Hello everybody. I don't really know where to look for anymore.
I've had this weird problem where, if i embed videos via iframe, i usually get the "Internal Server Error" message.
The videos can't be opened either via following the usualy link to the video. Same error message. 
I've tested many videos. Short ones seem to work fine but if they are longer than like 30 seconds, things just start to not work anymore.
Some help would be greatly appreciated. If more information is required, feel free to comment.

**Comment by CanEkizoglu** [comment]
Anyone?

**Comment by wolften** [comment]
Your URL must be in this format: https://www.youtube.com/embed/xxxx
If you try with the usual URL, it won't work. 
Of course, if you're trying to iframe Youtube videos, which I recommend over hosting them on your WikiJS server.
